integ-test/src/test/java/org/opensearch/sql/sql/RelevanceFunctionIT.java

```diff
@@ -5,33 +5,34 @@
 
 package org.opensearch.sql.sql;
 
+import static org.opensearch.sql.legacy.TestsConstants.TEST_INDEX_ACCOUNT;
 import static org.opensearch.sql.util.MatcherUtils.rows;
 import static org.opensearch.sql.util.MatcherUtils.schema;
 import static org.opensearch.sql.util.MatcherUtils.verifyDataRows;
 import static org.opensearch.sql.util.MatcherUtils.verifySchema;
 
 import java.io.IOException;
 import org.json.JSONObject;
-import org.junit.jupiter.api.Test;
+import org.junit.Test;
 import org.opensearch.sql.legacy.SQLIntegTestCase;
 
 public class RelevanceFunctionIT extends SQLIntegTestCase {
   @Override
   public void init() throws IOException {
-    loadIndex(Index.BANK);
+    loadIndex(Index.ACCOUNT);
   }
 
   @Test
-  void match_in_where() throws IOException {
-    JSONObject result = executeQuery("SELECT firstname WHERE match(lastname, 'Bates')");
+  public void match_in_where() throws IOException {
+    JSONObject result = new JSONObject(executeQuery("SELECT firstname FROM " + TEST_INDEX_ACCOUNT + " WHERE match(lastname, 'Bates')", "jdbc"));
     verifySchema(result, schema("firstname", "text"));
     verifyDataRows(result, rows("Nanette"));
   }
 
   @Test
-  void match_in_having() throws IOException {
-    JSONObject result = executeQuery("SELECT lastname HAVING match(firstname, 'Nanette')");
-    verifySchema(result, schema("lastname", "keyword"));
+  public void match_in_having() throws IOException {
+    JSONObject result = new JSONObject(executeQuery("SELECT lastname FROM " + TEST_INDEX_ACCOUNT + " HAVING match(firstname, 'Nanette')", "jdbc"));
```

Comment:
@forestmvey I just noticed that quite a few other classes have `executeJdbcRequest` method that does exactly that. `AggregationExpressionIT` is one example. 

Can you move it down to SQLIntegTestCase and use it here?

I think it make sense to update those other tests as well. You'll find all of them if you search for `executeJdbcRequest` symbol.

Reply:
Please see [Commit #2](https://github.com/Bit-Quill/opensearch-project-sql/pull/55/commits/cffd71ce8ffe4149227be7d9360bb7d64e2be0b8) for a resolution to this requested change.